### ISSUE TITLE
Enable visual regression tests on CI builds

### ISSUE THREAD

**Issue by phavekes** [OpenConext/OpenConext-engineblock#1681]
[This issue is imported from pivotal - Originaly created at May 1, 2019 by Michiel Kodde](https://www.pivotaltracker.com/story/show/165743005)

After all 5.11 chores/left overs have been addressed we can enable the visual regression tests on CI builds. 

As these tests are fragile by definition, it might be a good idea to mark them as an allowed failure.

Estimation: 1h

**Comment by phavekes** [comment]
Possibly include this in the 6.0 release? @bstrooband what do you think? Alternatively we can park this back into the ice box (Michiel Kodde - Sep 9, 2019)

**Comment by phavekes** [comment]
If 5.11 is considered stable as it\'s currently is I think we should enable them. We didn\'t enable them from the start to allow rapid development on the new error pages.
 Maybe we could enable them before we start developing a new release  so we could decide before the actual release is made if we want to keep them enabled. (bstrooband - Sep 10, 2019)

**Comment by phavekes** [comment]
We decided to postpone enabling this feature on the 6.0 EB release. The behavior on the dev vm and on Travis are too far apart, causing the regression tests from not passing. When we start using a more controlled environment on Travis (docker #168748949) we can retry. Leaving the PR open for now . (Michiel Kodde - Sep 25, 2019)
